src/reporter/report.screenshots.tpl.html

```diff
@@ -156,7 +164,11 @@
 <body>
 <div class="suites-menu">
   <% _.forEach(suites, function (suite, suiteIndex) { %>
-  <div class="name" onClick="selectSuite(<%- suiteIndex %>)"><%- suite.name %></div>
+    <% if (suite.statistic.specs.failed > 0) { %>
+    <div class="name failure" onClick="selectSuite(<%- suiteIndex %>)"><%- suite.name %></div>
```

Comment:
wouldn't it be easier to just add the "failure" style with inline expression ? This way you would avoid the code duplication ?

Reply:
Done, please take a look again.